browser/js/after-app-submit/after-app-submit.controller.js

```diff
@@ -0,0 +1,3 @@
+app.controller('afterSubmitCtrl', function ($rootScope) {
+  $rootScope.$broadcast('applicantView');
```

Comment:
This seems a little messy to me. I dug into your code—it looks like this is so that the navbar will hide for certain views. So your solution here does seem reasonable. But here are two thoughts for a different way you might be able to achieve the same end.
1) Names ui-views, [see here](https://github.com/angular-ui/ui-router/wiki/Multiple-Named-Views). These allows you to update multiple different `ui-view`s when a certain state loads.
2) You might also [use `urlRouterProvider.when`](http://angular-ui.github.io/ui-router/site/#/api/ui.router.router.$urlRouterProvider#methods_when) to catch certain urls.